packages/semantic-model/test/api-spec.ts

```diff
@@ -283,7 +284,7 @@ context("The ui5-language-assistant semantic model package API", () => {
         });
         // TODO: assert no cyclic references in extends or implements or parent - maybe not in a test
       });
-    });
+    }).timeout(GET_MODEL_TIMEOUT);
```

Comment:
if there was no timeout before, does it mean it was the default 2000ms?
if so what changed that requires x10 larger timeout?

Reply:
will check this again with the new proxy